### PR TITLE
feat: Option to remove registration of registered event. #168

### PR DESCRIPTION
**Pull request by nishant0708**
feat: Their should be option to delete created event by user. #167

-ADDED cancelregistration function in event cards page

#168 

https://github.com/Tanay-ErrorCode/lupo-skill/assets/101548649/09a963db-8254-44f9-a991-173ad46a08e1

### CODE REVIEW

**Comment by nishant0708** [comment]
hey @Tanay-ErrorCode @Doraemon012 let me know any changes if required

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Keep the **register button** where it was initially. If the user is registered for the event, then the button should change to cancel registration button, which should be red.

**Comment by nishant0708** [comment]
Actually I tried doing that let me try again. Actually I was facing error in error in making card responsive so I made it like that

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Position it where it was previously, just change the color and the text if user is registered. 
You can make the text **"Unregister"**

**Comment by nishant0708** [comment]
@Tanay-ErrorCode @Doraemon012  check it now 
Screenshot 2024-06-03 122009

**Comment by nishant0708** [comment]
hey @Tanay-ErrorCode i am really sorry i mistakenly made pr for issue #168 for which i was not assigned . I am really sorry for that

**Comment by Doraemon012** [comment]
@nishant0708 No problem. Please proceed with the PR.

**Comment by nishant0708** [comment]
@Doraemon012 thanks. i have updated the required changes . please let me if there is need of any change.

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Please fix the hover color.
image

**Comment by nishant0708** [comment]
Hey @Tanay-ErrorCode Please Check It Now.

**Review by Doraemon012 — APPROVED** [review]
LGTM

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Good work!
